src/components/Buttons.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Buttons({ onClickHandler, isRequested }) {
```

Comment:
`Button`, or as it is not general button component but specific only for request sending can be named `LoadDataButton` etc.